spec/models/manageiq/providers/microsoft/infra_manager/refresher_spec.rb

```diff
@@ -5,22 +5,10 @@
     FactoryGirl.create(:miq_region)
     @ems = FactoryGirl.create(:ems_microsoft_with_authentication, :zone => zone,
         :hostname => "scvmm1111.manageiq.com", :ipaddress => "192.168.252.90", :security_protocol => "ssl")
-    data_file = File.join(File.dirname(__FILE__), %w(.. .. .. .. .. tools scvmm_data get_inventory_output.yml))
-
-    # See HACK comment in EmsMicrosoft.raw_connect for details around suppressing
-    # the benign GSSAPI warnings printed when the winrm gem is required.
-    # These warnings are also printed when the webmock gem
-    # is required by rspec since it also depends on the GSSAPI gem too.
-    require 'winrm'
-
-    # Match the output returned by winrm with our compressed output.
-    output = WinRM::Output.new
-    output.merge!(YAML.load_file(data_file))
-    temp = output.stdout
-    output[:data].map{ |h| h.delete(:stdout) }
-    output[:data][0][:stdout] = Base64.encode64(ActiveSupport::Gzip.compress(temp))
 
-    allow_any_instance_of(WinRM::WinRMWebService).to receive_messages(:run_powershell_script => output)
+    data_file = File.join(File.dirname(__FILE__), %w(.. .. .. .. .. tools scvmm_data get_inventory_output.yml))
```

Comment:
Prefer using a path relative to Rails.root... something like `Rails.root.join("spec/tools/scvmm_data/get_inventory_output.yml")`